### PR TITLE
make pr_diff.sh allow timestamp querry

### PR DESCRIPTION
**Pull request by favadi**

### CODE REVIEW

**Comment by favadi** [comment]
[Monthly core development progress report (2018-June)](https://trello.com/c/skIKckJe/160-monthly-core-development-progress-report-2018-june)

File: pr_diff.sh [hunk 1]
```diff
@@ -1,17 +1,49 @@
 #!/bin/bash
```
Comment (vietthang207) [inline pr_diff.sh]: What is the purpose of this script? From my understanding, it just add an url of the pull request?

File: pr_diff.sh [hunk 2]
```diff
@@ -1,17 +1,49 @@
...
+	;;
+    "*")
+	print_usage
+	exit 1
+esac
 
 for commit_id in "${pr_commit_ids[@]}"; do
     git show --format="%B" "$commit_id" |\
-	awk -v github_url="$github_url" 'NR == 1 {$4="["$4"]("github_url substr($4,2)")"; print "* "$0; next}; {print "  "$0}'
+	awk -v github_url="$github_url" 'NR == 1 {$4="["$4"]("github_url substr($4,2)")"; print "* "$0; next}; /^diff --/{exit}; {print "  "$0}'
```
Comment (vietthang207) [inline pr_diff.sh]: I seems to me that "git show" will not show any diff message, do you have an example of such an output?

Comment (vietthang207) [reply pr_diff.sh]: I've try "git show -m commit_ids", it will show some diff if the commit is a merge commit.

Comment (favadi) [reply pr_diff.sh]: > I seems to me that "git show" will not show any diff message, do you have an example of such an output?

You could just run git show in any of your repos.

File: pr_diff.sh (hunk 1 above)
Comment (favadi) [reply pr_diff.sh]: To generate a changelog from git history.

**Pull request merged**